src/pages/Scholarships.jsx

```diff
@@ -21,11 +24,23 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 function ScholarshipsPage() {
+  const classes = useStyles();
   const [scholarships, setScholarships] = useState([]);
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState();
   const [sortField, setSortField] = useState('deadline');
   const [sortDir, setSortDir] = useState('asc');
+  const history = useHistory();
+  const location = useLocation();
+  const setFoo = (val) => {
+    history.push({
+      search: queryString.stringify({
+        ...queryString.parse(location.search),
+        sorting: val,
+      }),
+    });
+  };
+  const foo = queryString.parse(location.search).foo ?? DEFAULT_SORT_URL_VALUE;
```

Comment:
```suggestion
  const sortDir = queryString.parse(location.search).sortDir ?? DEFAULT_SORT_DIR;
```

Reply:
TODO: Make sure the sortDir and sortField values make sense. Otherwise use the default values.